src/common/hotbar-store.ts

```diff
@@ -175,18 +153,24 @@ export class HotbarStore extends BaseStore<HotbarStoreModel> {
       return;
     }
 
-    if (cellIndex == -1) {
+    if (cellIndex === undefined) {
       // Add item to empty cell
-      const emptyCellIndex = hotbar.items.findIndex(isNull);
+      const emptyCellIndex = hotbar.items.indexOf(null);
 
       if (emptyCellIndex != -1) {
         hotbar.items[emptyCellIndex] = newItem;
       } else {
-        // Add new item to the end of list
-        hotbar.items.push(newItem);
+        if (ipcRenderer) {
+          import("../renderer/components/notifications")
```

Comment:
This is needed because it doesn't exist on `main`. I really wish we would split these into main and renderer sides.

Reply:
I think we should just broadcast a message and act on that within renderer. Would keep store cleaner.

Reply:
Okay, it just seems like hotbar-store is really a renderer-only thing but I can change it.